CONTRIBUTING.md

```diff
@@ -0,0 +1,213 @@
+<h2>📝&nbsp; Table of Content</h2>
```

Comment:
this whole CONTRIBUTING file, I couldn't find how to access it

is it accessible only via file in the github repo?

if you get to [wallet repo home](https://github.com/FuelLabs/fuels-wallet), we have a link to also a "Contributing" section.

Looks like there're some dupe/conflicting info between both Contribution sections.

Can we change the doc in other PR and add only doc related to feature to link package with `fuel-ui` ?

Reply:
would be nice to add a new section in our doc including this info to link with fuel-ui


<img width="350" alt="image" src="https://user-images.githubusercontent.com/8636507/215006424-a1f516a9-16c1-4cb7-bd1f-5c0e4d07c966.png">

Reply:
also can we revert `pnpm-lock` changes as was not needed to change dependency versions

Reply:
You are right, I totally forgot we have a documentation right now 😅